ProbFX-RoadMarkings.cabal

```diff
@@ -1,50 +1,43 @@
 cabal-version:      2.4
 name:               ProbFX-RoadMarkings
-version:            0.1.0.0
+version:            0.1.2.0
 
--- A short (one-line) description of the package.
--- synopsis:
-
--- A longer description of the package.
--- description:
+synopsis: Road Lane detection using ProbFX
+description:
+    Using ProbFX for a probabilistic model and OpenGL for rendering this project
+    aims to automate the labelling of basic road markings on dashcam images.
 
 -- A URL where users can report bugs.
--- bug-reports:
+bug-reports: https://github.com/Functional-Labelling-Lab/ProbFX-RoadMarkings/issues
 
--- The license under which the package is released.
 -- license:
-author:             God Emperor
+author:             Functional Labelling Lab
 maintainer:         bcieslar2001@gmail.com
 
--- A copyright notice.
--- copyright:
--- category:
-extra-source-files: CHANGELOG.md
+build-type: Custom 
+
+custom-setup
+  setup-depends: base, Cabal, process, directory
+
+extra-source-files:
+    backend/src
 
 library
     exposed-modules:  CppFFI
+    build-depends:    base, c-storable-deriving, containers
+    hs-source-dirs:   src/CppFFI
 
-    -- Modules included in this library but not exported.
-    -- other-modules:
+    extra-libraries: 
+    RoadMarkingsBackend, glad, glfw3,
+    /usr/lib/x86_64-linux-gnu/librt.so,
```

Comment:
https://stackoverflow.com/questions/7695638/conditional-cmake-link-to-rt-library

We may be able to put `rt` here, this is currently failing the CI